src/os/macos.rs

```diff
@@ -22,6 +22,13 @@ pub trait WindowExt {
     /// - `false`: the dock icon will only bounce once.
     /// - `true`: the dock icon will bounce until the application is focused.
     fn request_user_attention(&self, is_critical: bool);
+
+    /// Toggles a fullscreen mode that doesn't require a new macOS space.
+    ///
+    /// This is how fullscreen used to work on macOS in versions before Lion.
+    /// And allows the user to have a fullscreen window without using another
+    /// space or taking control over the entire monitor.
+    fn set_simple_fullscreen(&self, enable: bool);
```

Comment:
We usually try to give parameters specific names, i.e. `fullscreen` instead of `enable`.